### ISSUE TITLE
Reorganization of the Schedule

### ISSUE THREAD

**Issue by nguy** [gamaanderson/2017-AMS-Short-Course-on-Open-Source-Radar-Software#2]
This follows with previous short course schedules that the Py-ART and BALTRAD teams have put together. I've attended a couple of these and they have been great.

I'd like to propose a change in the schedule to see what the team thinks. What if we flip the model around - do some demo and interaction with the tools at the beginning of the day to show the usefulness of these packages? We could then follow the demos with some more in depth discussion of the development and "nuts and bolts" of the packages, along with some general python pointer. Finally ending with an update from current ongoing projects (e.g. LROSE) and question/discussion. 

Any thoughts on this reorg? I feel like engaging the audience first and allowing them to continue to explore the packages while we dive into the architecture could be useful.

**Comment by gamaanderson** [comment]
I agree, the "using artview" section can be put in the beginning just after introduction as it needs no programming. Also some pyart example could be presented. 

@DanielMichelson, Could BALTRAD use an introduction and a later advanced section?

**Comment by DanielMichelson** [comment]
Thanks @gamaanderson for referencing #1. I've already given my opinion there on the order of things, so I'll let others chip in their views.
